lib/msal-browser/test/telemetry/BrowserPerformanceClient.spec.ts

```diff
@@ -1,4 +1,14 @@
-import { PerformanceEvents } from "@azure/msal-common";
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    ApplicationTelemetry,
+    Logger,
+    PerformanceEvents,
+} from "@azure/msal-common";
+import { name, version } from "../../src/packageMetadata";
```

Comment:
Reason for additional imports?

Reply:
Which ones?

Reply:
All of them, looks like adding imports is the only change in this file, want to understand why